pkg/utils/builders.go

```diff
@@ -116,7 +116,7 @@ func BuildingAndLaunchJob(experiment *ExperimentDetails, clients ClientSets) err
 
 // launchJob spawn a kubernetes Job using the job Object received.
 func (experiment *ExperimentDetails) launchJob(job *batchv1.Job, clients ClientSets) error {
-	_, err := clients.KubeClient.BatchV1().Jobs(experiment.Namespace).Create(job)
+	_, err := clients.KubeClientExperiment.BatchV1().Jobs(experiment.Namespace).Create(job)
```

Comment:
Just trying to understand which config is used for which purpose.
- kubeconfig & KubeClientExperiment for ChaosEngine, ChaosExperiment, ChaosRunner, ChaosOperator
- litmuskubeconfig & k8sClientSetLitmus for TargetApplication, ExperimentJob/Pod, ChaosResult
right? if this is the case, would you like to use the `clients.KubeClientExperiment` to get the chaosresult verdict [here](https://github.com/litmuschaos/chaos-runner/blob/49f4df2d244fe0a95fd4e6696f85e1fff08c7737/pkg/utils/watchJob.go#L65)?
.

Reply:
Well I was thinking having the chausresult used by kubeconfig. Do you think it should be used by litmuskubeconfig?